_starter-kit/_patterns/02-layouts/grids/_grid.scss

```diff
@@ -0,0 +1,92 @@
+// @file
+// Styles for the grid layout.
+$bp-grid-small: 700px;
+$bp-grid-large: 1000px;
+$bp-grid-mega: 1300px;
+
+$grid-item-minwidth: 400px;
+$grid-gutter: $layout-gutter-width;
+
+.l-grid-3col {
```

Comment:
should these styles be reorganized to use variants (e.g., l-grid, l-grid--3col, l-grid--4col)?

Reply:
I'll consolidate the initial grid set up with a placeholder. But I'm not sure who much is gained by further consolidation and variant overrides. I can see the potential for a l-grid-[#]col--[modifier] variant where you modify a single or group of cells in the grid. But I think we should leave the column # as separate entities.